### ISSUE TITLE
Resource Hog

### ISSUE THREAD

**Issue by DerekGooding** [FlorianButz/DynamicWin#5]
I'll have to look into why, but the CPU usage for a background application is high. Also it garbage collects often. 

My immediate guess is it has something to do with handling the lifetime of objects poorly. Or possibly the 1 second running tasks could be cached and only handled on changes.

**Comment by FlorianButz** [comment]
Yes, my code is probably very poorly optimized especially for a background application. I am not very experienced in making optimal high efficiency code.

**Comment by DerekGooding** [comment]
I'm noticing you're using your own OnDestroy methods. I'd consider looking into the IDisposable interface and using it instead. 

If you don't mind me contributing, I can try to clean some things up for you. Do you consider this project done? Or are you looking to expand on it? Totally get it if this is the final release. I won't push.

**Comment by FlorianButz** [comment]
The project is not done since there are still a few things I want to get done, however I won't work as much on it as before release. I would love to see you contribute and helping the project, definetley!
The main reason I think the CPU is used so much is because of the rendering. The app renders at ~60fps at all time and does not use hardware acceleration (I think). And I couldn't get OpenGL to work with SkiaSharp, otherwise it'd probably also run faster on more modern hardware.

**Comment by DerekGooding** [comment]
I'm concerned this adds itself to startup invisibly and is a high impact on startup. 

This is dangerous functionality and could get you flagged for malware. At the minimum, it needs to show itself in the tray. Functioning invisibly while having a passive drain the the PC is a huge problem. Consider fixing it before someone reports you for malicious software.

**Comment by FlorianButz** [comment]
I will add a tray icon as soon as possible. Thanks for letting me know

**Comment by FlorianButz** [comment]
I added the tray icon and an option to enable the app on system startup. This option is false by default in order to avoid confusion.

**Comment by FlorianButz** [comment]
Better resource management in V2, especially RAM.
